### PR TITLE
Compose project name

### PR DESCRIPTION
**Pull request by jspaaks**
I ran into trouble relating to name clashes when testing RSD instances for multiple organizations on one machine. Name clashes can happen with images, containers, networks, and volumes. For images and containers you can circumvent the problem by using env vars (even with fallback behavior and error raising if you want, see https://docs.docker.com/compose/compose-file/#variable-substitution), but volumes are more problematic. Basically, you cannot prepend them with your own variables, but it will include ``COMPOSE_PROJECT_NAME`` if that's been defined.

Long story short, this PR uses a combination of
- ``COMPOSE_PROJECT_NAME`` from the ``.env`` file to prepend the volume names and the network
- environment variables with fallback syntax (``${COMPOSE_PROJECT_NAME-rsd-}``) to name the images and containers.

This has the happy side effect of no longer needing the --project-name syntax when doing e.g. 

```
docker-compose up
```

~~Travis fails but I think that has to do with timeouts or something.~~

### CODE REVIEW

**Comment by sverhoeven** [comment]
I like the side-effect.

Tried it and worked as documented and expected.
